experiences/homepage/homepage.test.js

```diff
@@ -52,8 +51,54 @@ const homepageTest = function() {
 		expect(cardExists).equal(true);
 	});
 
+	it('scroll bounded', async function() {
+		// Initial 300px scroll to trigger
+		// First 60px transitioner doesn't made the card appear
+		const initialBrowserScroll = 300 + 60;
+		const smallScrollPixels = 20;
+		const cardSelector = config.cards.homepage.cardSelector;
+		const scrollBoundedSnapPoints = [];
+		const timeout = 150;
```

Comment:
with the default timeout of 100 tests were not working (different values when putting breakpoints or not).